homeassistant/components/home_connect/api.py

```diff
@@ -69,7 +71,7 @@ def get_devices(self):
             elif app.type == "Hob":
                 device = Hob(self.hass, app)
             else:
-                _LOGGER.warning("Appliance type %s not implemented", app.type)
+                _LOGGER.warning("Appliance type %s not implemented.", app.type)
```

Comment:
We don't end logging messages with period.

Reply:
Period removed.